e2e_tests/test_search.py

```diff
@@ -149,3 +176,87 @@ def test_click_on_clear_all_as_standard_user_redirects_to_browse_transferring_bo
         standard_user_page.get_by_role("link", name="Clear all").click()
 
         expect(standard_user_page).to_have_url(url)
+
+
+class TestOpenSearchResults:
+    @property
+    def search_route_url(self):
+        return "/search_results_summary"
+
+    @property
+    def browse_transferring_body_route_url(self):
+        return "/search/transferring_body"
+
+    @property
+    def transferring_body_id(self):
+        return "c3e3fd83-4d52-4638-a085-1f4e4e4dfa50"
+
+    def test_test_search(self, aau_user_page: Page):
```

Comment:
why do we have test_test rather than just test? (here and the test function below)